src/Turbo/README.md

```diff
@@ -288,8 +289,8 @@ class ChatController extends AbstractController
     <div id="messages" {{ turbo_stream_listen('chat') }}>
         {#
             The messages will be displayed here.
-            "turbo_stream_listen()" automatically registers a Stimulus controller that subscribes to the "chat" Mercure topic using EventSource.
-            The connection to the Mercure Hub is automatically closed when this HTML block is removed.
+            "turbo_stream_listen()" automatically registers a Stimulus controller that subscribes to the "chat" topic of the transport.
```

Comment:
This should be reverted. This works for Mercure, but it's implementation specific and some transports may not use a topic mechanism at all.
Also giving details about how it works with Mercure (using EventSource) will improve the debugging experience (you know where to look in Chrome's dev tools).

Reply:
"topic" is a mandatory argument of the `turbo_stream_listen()`, so it cannot be transport-specific, at least not at this level (internally yes, they could map this to another word for a similar concept)
I think the doc should be as decoupled from Mercure as possible, no prevent any confusion in ppl's minds. Mercure is a possible and nicely working implementation, not the only possible one.
About SSE, this readme is not the place, "see Mercure's doc for more"